### ISSUE TITLE
Errors in tsc are not passed through to console in ember ts:precompile

### ISSUE THREAD

**Issue by BryanCrotaz** [typed-ember/ember-cli-typescript#917]
ember-cli: 3.11.0
node: 10.15.0
os: darwin x64

tsc: Version 3.6.4

yarn list v1.19.0
├─ ember-cli-typescript-blueprints@3.0.0
├─ ember-cli-typescript@3.0.0
└─ ember-load-initializers@2.1.0
   └─ ember-cli-typescript@2.0.2

### What are instructions we can follow to reproduce the issue?
1. Create new ember typescript addon
2. Declare a variable but don't use it

```
yarn publish

$ ember ts:precompile
Command failed with exit code 2 (ENOENT): tsc --allowJs false --noEmit false --rootDir /Users/bryan/Projects/ember-cli-template-editor --isolatedModules false --declaration --declarationDir /var/folders/24/9c9x7j316gbd3z5wg84fghqc0000gn/T/e-c-ts-precompile-22635 --emitDeclarationOnly


Stack Trace and Error Report: /var/folders/24/9c9x7j316gbd3z5wg84fghqc0000gn/T/error.dump.86bf8ef41ca4361feee2af2c0d3bdf1e.log
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/publish for documentation about this command.



Bryans-MacBook-Pro:ember-cli-template-editor bryan$ tsc
addon/classes/PsdImporter.ts:74:7 - error TS6133: 'clipped' is declared but its value is never read.

74   let clipped = layer.clipped;
         ~~~~~~~


Found 1 error.
```

Expected: `ts:precompile` output shows the tsc output so that debugging is easy

**Comment by chriskrycho** [comment]
Thanks for reporting this! Definitely could use to improve the dev experience here!

**Comment by jamescdavis** [comment]
Resolved by #984 

Released in [v3.1.2](https://github.com/typed-ember/ember-cli-typescript/releases/tag/v3.1.2)
